src/components/Navbar/SettingsDrawer/ReciterSelectionBody.tsx

```diff
@@ -59,22 +59,28 @@ const SettingsReciter = () => {
             : data.reciters;
           return (
             <div>
-              {filteredReciters.map((reciter) => (
-                <label className={styles.reciter} htmlFor={reciter.id.toString()} key={reciter.id}>
-                  <input
-                    id={reciter.id.toString()}
-                    type="radio"
-                    name="reciter"
-                    value={reciter.id}
-                    checked={reciter.id === selectedReciter.id}
-                    onChange={(e) => {
-                      onSelectedReciterChange(e.target.value, data.reciters);
-                    }}
-                  />
-                  <span>{reciter.name}</span>
-                  <span className={styles.recitationStyle}>{reciter.style.name}</span>
-                </label>
-              ))}
+              {filteredReciters
+                .sort((a, b) => (a.name > b.name ? 1 : -1))
```

Comment:
This will run on every render, might be cleaner to do do the sorting and mapping in a useMemo